WorkspaceServer/Packaging/PackageInitializer.cs

```diff
@@ -58,6 +61,17 @@ public class PackageInitializer : IPackageInitializer
                 await afterCreate(directory, budget);
             }
         }
+
+        private static string GetLanguageFromProjectName(string projectName)
```

Comment:
>string [](start = 23, length = 6)

A more constrained type would be a good idea.

Reply:
If I'm understanding this correctly, once #234 is in we can modify the `WorkspaceRequest` object to contain a language name which would then allow the removal of the `WorkspaceServerMultiplexer` and by extension this method, because we'd then have access to the language at the `Workspace` level and it could then be plumbed through where appropriate.  E.g., if `WorkspaceRequest.Workspace.Language` was a valid value, then [this line](https://github.com/dotnet/try/blob/9e4d03378204b040dddfa0939b5a56fb7523ca49/MLS.Agent/Controllers/LanguageServicesController.cs#L130) would be able to appropriately select `RoslynWorkspaceServer` or `FSharpWorkspaceServer`.